File: Hishab Server/hishabApp/hishabApp/src/main/java/com/example/hishab/hishabApp/apiModels/ProductResponse.java
package com.example.hishab.hishabApp.apiModels;

import com.example.hishab.hishabApp.model.Product;

import lombok.AllArgsConstructor;
import lombok.Data;


@Data
@AllArgsConstructor
public class ProductResponse {
	Product product;
	int localId;
	CategoryResponse categoryResponse;
}

File: Hishab/app/src/main/java/com/example/hishab/retrofit/response/ProductResponse.kt
package com.example.hishab.retrofit.response


import com.example.hishab.models.entities.Product
import com.example.hishab.retrofit.response.CategoryResponse

class ProductResponse() {
    var productId = 0
    var productName = "";
    var localId = 0
    var categoryResponse: CategoryResponse? = null

}